### ISSUE TITLE
Create a develop branch

### ISSUE THREAD

**Issue by JonathanKessler** [VarenTechInternship/varentech-deploya#12]
Now that we have an intial commit, and well done!, we should only push completed releases to the master branch. Moving forward we should be working out of a develop branch. Each ticket that requires changes to the repository should get its own branch, cloned from develop. When that work is done and tested, it will be merged back to develop. Releases will be tagged in develop and merged back to master.

Long story short, please create a branch named develop that is a clone of master.

**Comment by kesslerk** [comment]
That makes sense. The branch is created.
